script/console

```diff
@@ -23,6 +23,20 @@ if [ -n "$1" ]; then
 else
   # no argument provided, so just run the local console in the development
   # environment. Ensure the application is up to date first.
-  script/update.sh
-    echo "local console not yet implemented"
-fi
\ No newline at end of file
+  # script/update.sh
+
+  echo "To run a local console, please execute the following command:"
+  echo "    source ~/python3.6/bin/activate"
```

Comment:
Not entirely sure what this script is used for, but should it actually be running the `source ~/python3.6/bin/activate` statement rather than just printing it?